mozilla-release/mobile/android/base/java/org/mozilla/gecko/BrowserApp.java

```diff
@@ -1942,12 +1942,13 @@ public void run() {
                         }
                     }
                 });
-
+                /* Cliqz Start */
                 // Display notification for Mozilla data reporting, if data should be collected.
-                if (AppConstants.MOZ_DATA_REPORTING &&
-                        Restrictions.isAllowed(this, Restrictable.DATA_CHOICES)) {
-                    DataReportingNotification.checkAndNotifyPolicy(this);
-                }
+//                if (AppConstants.MOZ_DATA_REPORTING &&
+//                        Restrictions.isAllowed(this, Restrictable.DATA_CHOICES)) {
+//                    DataReportingNotification.checkAndNotifyPolicy(this);
+//                }
+                /* Cliqz End */
```

Comment:
See  the [Coding Style](https://github.com/cliqz-oss/cliqz-android/wiki/Coding-style) guide